addons/website/static/src/systray_items/new_content.js

```diff
@@ -170,8 +171,15 @@ export class NewContentModal extends Component {
     }
 
     createNewPage() {
+        if (this.isCreatingNewPage) {
+            // Prevent opening the "New page" dialog several times.
+            return;
+        }
```

Comment:
I cannot reproduce the initial issue this was solving. In short, at some point, the opening of the dialog was delayed until the response from `get_new_page_templates` was obtained - and this was used to prevent additional click from opening more instances of the dialog.
I removed the associated code in this commit: <https://github.com/odoo/odoo/pull/138748/commits/78a299166574a5a59f3d5df01b8872a760e560aa>